assets/src/apps/scheduler/PageScheduleLine.tsx

```diff
@@ -36,7 +36,7 @@ export const PageScheduleLine: React.FC<ScheduleLineProps> = ({ item, indent, da
       let targetStartDate: Date | DateWithoutTime | null = startDate;
 
       if (item.startDateTime && startDate) {
-        targetStartDate = new Date();
+        targetStartDate = new Date(2024, 1, 1);
```

Comment:
Is there a specific reason this date is hard coded? Or is it just a starting date that is required and will always be overridden so the value doesn't actually matter here?

Reply:
I missed this one when I changed it to the 3-param version of setFullYear.

But you will find one like this later because DateWithoutTime doesn't have a 3-param version of setFullYear and we needed to start with a 31 day month.